internal/api/api.go

```diff
@@ -138,12 +138,14 @@ func (server *Server) serveReads(w http.ResponseWriter, req *http.Request) {
 	}
 
 	if region.End > 0 && region.Start > region.End {
-		writeError(w, newInvalidRangeError(fmt.Errorf("%s: start > end", region)))
+		writeError(w, newInvalidRangeError(fmt.Errorf("%v: start > end", region)))
```

Comment:
Why did you change from %s to %v?  %s is correct here...

Reply:
the tests would not run with %s. I got a type mismatch error

Reply:
You're right - I thought Region had a String() method.  This is something new being caught by go 1.10.  Please make a new change (before this one) that adds a String method for region (since the value itself won't be very helpful).

There's a related error caught by the 1.10 compiler in htsget-client/main.go:
main.go:130: Fatalf call needs 1 arg but has 2 args